telco_ref_design_specs/ran/telco-ran-ref-du-crs.adoc

```diff
@@ -0,0 +1,39 @@
+:_content-type: ASSEMBLY
+[id="telco-ran-ref-du-crs"]
+= Telco RAN DU reference configuration CRs
+include::_attributes/common-attributes.adoc[]
+:context: ran-ref-design-crs
+
+toc::[]
+
+Use the following custom resources (CRs) to configure and deploy {product-title} clusters with the telco RAN DU profile.
+
+[NOTE]
+====
+Some of the CRs are optional depending on your requirements.
+
+CR fields that you can change are annotated in the CR with YAML comments.
+====
+
+include::modules/telco-ran-crs-day-2-operators.adoc[leveloffset=+1]
+
+include::modules/telco-ran-crs-cluster-tuning.adoc[leveloffset=+1]
+
+include::modules/telco-ran-crs-machine-configuration.adoc[leveloffset=+1]
+
+[id="telco-reference-ran-du-use-case-yaml_{context}"]
+== YAML reference
+
+The following is a complete reference for all the custom resources (CRs) that make up the telco RAN DU {product-version} reference configuration.
+
+[NOTE]
+====
+You can extract the YAML CRs from the `ztp-site-generate` container.
+For more information, see link:https://docs.openshift.com/container-platform/latest/scalability_and_performance/ztp_far_edge/ztp-preparing-the-hub-cluster.html#ztp-preparing-the-ztp-git-repository_ztp-preparing-the-hub-cluster[Preparing the {ztp} site configuration repository].
```

Comment:
Make this link an xref

Reply:
This has to be a link because the docs are being delivered in a separate distro unfortunately.